src/xma/include/lib/xmahw_lib.h

```diff
@@ -40,6 +40,8 @@
 #define MAX_KERNEL_REGMAP_SIZE    4032//Some space used by ert pkt
 #define MAX_REGMAP_ENTRIES        1024//Int32 entries; So 4B x 1024 = 4K Bytes
 
+const uint64_t mNullBO = 0xffffffff;
```

Comment:
At one point when we port to Windows this will bite you.  If this is meant to refer to a BO handle returned from driver, then can you replace with NULLBO as defined in xrt.h?

Reply:
done